src/main/java/com/artipie/rpm/meta/MergedXmlPrimary.java

```diff
@@ -132,6 +129,8 @@ public Result merge(final Map<Path, String> packages, final Digest dgst, final X
         boolean valid = true;
         final Collection<String> res = new ArrayList<>(locations.size());
         String checksum = "123";
+        reader.nextEvent();
+        reader.nextEvent();
```

Comment:
These lines look not clear: why do we need to skip two events here? I think we need to document it somehow.